File: spec/fixtures/test_project/SiblingProjects/SiblingProjects/ViewController.m
//
//  ViewController.m
//  SiblingProjects
//
//  Created by Ilya Dyakonov on 14.09.2022.
//

#import "ViewController.h"
#import "StaticLibrary.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UILabel *label;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.label.text = [[StaticLibrary new] something];
}


@end

File: spec/fixtures/test_project/StaticLibModules/StaticLibModules/ViewController.swift
//
//  ViewController.swift
//  StaticLibModules
//
//  Created by Ilya Dyakonov on 4/7/20.
//  Copyright © 2020 xcode-archive-cache. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet var reportedClassNameLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        reportedClassNameLabel.text = ClassNameReporter.reportedClassName()
    }
}

